PEP 588 Updates (#1139)
python/peps: pep-0588.rst

in order to migrate bug tracking to GitHub, with the least impact on
CPython developer productivity.

Hire a professional project manager
-----------------------------------

Having a professional project manager to handle the migration, similar to how
the Warehouse project was managed, would help ensure the success of this project.

Create a playground CPython issue tracker on GitHub
---------------------------------------------------

We should create a playground issue tracker on GitHub where we can experiment

> test out the new workflow.

Reviewer: ```suggestion
and test out the new workflow.
```

<https://discuss.python.org/t/proposal-create-bug-triage-team-on-github/992/5>`_
for us to create a "bug triage" team on GitHub to help with closing
issues, notifying the appropriate parties, as well as applying labels
to issues and pull requests.

The new Triage role on GitHub is currently in beta, and the Python organization
has been granted access to this role, and we can begin taking advantage of it.

> The "Python Triage" team has been created. Description and expectation of the

Reviewer: ```suggestion
The "Python Triage" team has been created. A description and expectations of the
```

> triage role `has been added to Devguide <https://devguide.python.org/triaging/#python-triage-team>`_.

Reviewer: ```suggestion
triage role `have been added to Devguide <https://devguide.python.org/triaging/#python-triage-team>`_.
```
